Objects/codeobject.c

```diff
@@ -1246,7 +1246,7 @@ PyTypeObject PyCode_Type = {
 int
 PyCode_Addr2Line(PyCodeObject *co, int addrq)
```

Comment:
I would prefer to treat the second argument as an instruction offset, rather than a byte offset. Rather than changing PyCode_Addr2Line() calls. It sounds unfortunate that the API allows to ask the address in the middle of a word (2 bytes).

It sounds common to pass f_lasti to PyCode_Addr2Line(): PyCode_Addr2Line(code, f_lasti) would still work, no?

Reply:
Depends on how you think the API is defined.
If you want the the line number from an offset derived from `code.co_linetable`, then the API should be unchanged.

Reply:
`PyCode_Addr2Line(code, offset)` must take a *byte* offset. 
To get`f_lasti` from the frame, the only API to do so is something like `PyObject_GetAttrString(frame, "f_lasti")`, which would return a byte offset. 

Also see:
https://www.python.org/dev/peps/pep-0626/#c-api
https://www.python.org/dev/peps/pep-0626/#out-of-process-debuggers-and-profilers